README.md

```diff
@@ -38,12 +38,13 @@ A pre-built image for the web app is on Docker Hub as [nein09/polder-federated-s
 Assuming that you're starting from **this directory**:
 1. To build and run the web app, Docker needs to know about some environment variables. There are examples ones in `dev.env` - copy it to `.env` and fill in the correct values for you. Save the file and then run `source .env`.
 1. Install [Docker](https://docker.com)
-1. `cd docker`
-1. `docker-compose up -d`
-1. Go to your [local Blazegraph instance](http://localhost:9999/blazegraph/#namespaces) and add a new namespace - this is because the default one does not come with a full text index. Name it `polder` (or whatever you want, but you will need to change the GLEANER_ENDPOINT_URL environment variable if you don't name it that). Select 'quads' in the mode dropdown menu, and check the text boxes after "Full text index" and "Enable geospatial".
-1. If you want to try queries out on Blazegraph directly, be sure to click 'Use' next to your new namespace after you create it.
-1. `docker-compose --profile setup up -d` in order to start all of the necessary services and set up Gleaner for indexing.
-1. Do a crawl (these instructions assume you are in the `docker` directory):
+    - Also, for windows, [click here](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi) to install the WSL (windows subsystem for linux)
+3. `cd docker`
```

Comment:
If you leave these numbers all as "1.", then Markdown will automatically turn them into the correct things, so someone can add another one in the middle or take one out without having to  renumber all of them.